### DISCUSSION TITLE
Sudden "Loading chunk 0 failed" for dynamic components

### DISCUSSION THREAD

**Discussion by eric-burel** [vercel/next.js#17162] [Help]
Hi, 
I suddenly started to get this error when using a dynamically imported component: 
```
Loading chunk 0 failed. (missing: http://localhost:3000/_next/static/chunks/0.js)
ChunkLoadError: Loading chunk 0 failed. (missing: http://localhost:3000/_next/static/chunks/0.js) at Function.requireEnsure [as e] (http://localhost:3000/_next/static/runtime/webpack.js?ts=1600327016776:914:26) at Function.fn.e (http://localhost:3000/_next/static/runtime/webpack.js?ts=1600327016776:177:40) at _c (webpack-internal:///./src/components/map/AssetsMap/AssetsMapContainer.tsx:55:30) at
```
`AssetsMapContainer` being the component that calls a dynamic component like so:
```
const AssetsMap = dynamic(() => import("./AssetsMap"), {
  ssr: false
});
```

The weird thing is that this issue appeared like 2 weeks ago, while this component is months old. I didn't update Next (v9.4.4) nor Webpack (v4.43.0).

Did someone hit the same issue? Should I update Next to 9.5?

**Comment by fn-gabriel** [comment]
I'm getting this issue but I'm not even loading any dynamic components. I get this issue even if I do `yarn create next-app` and run `yarn dev` on a new project. It only happens on Next 9.5 though. Next 9.4 is fine for me. I tried multiple versions of Node too (12, 13, and 14).

**Reply by fn-gabriel** [reply]
@eric-burel what browser are you using? This was happening to me on Brave but on any other browser it works fine. May be something with a recent update.

**Reply by eric-burel** [reply]
Yeah it definitely smells like an underlying library updating. I use Chrome. I'll try again on monday.

**Comment by eric-burel** [comment]
It seems to affect Vulcan Next also, I'll try to have a more consistent repro, while I use a more recent version and no dynamic component. Very weird.
